commands/info.md

```diff
@@ -272,6 +272,8 @@ Here is the description of fields.
 Here is the meaning of all fields in the **server** section:
 
 *   `redis_version`: Redis OSS version this Valkey server is compatible with
+*   `valkey_version`: Valkey full version number (e.g. 7.2.5)
+*   `valkey_release_stage`: The release status reflects the Valkey server version release stage: "dev" in unstable, "rc1..N" during release, and "ga" when finalized
```

Comment:
Oops, we forgot to add `valkey_version` last year.

Should we clarify that `valkey_version` doesn't include suffixes like "-rc1"? Maybe skip the word "full" in "full version number"?

Let's describe release stage in a user perspective rather than dev/maintainer perspective. Mention "ga" first, don't mention the unstable branch name, etc.

```suggestion
*   `valkey_version`: Valkey version number (e.g. 7.2.5)
*   `valkey_release_stage`: The status of the Valkey version: "ga" for generally available versions; "rc1", "rc2", etc. for release candidates; "dev" for development versions
```